ts/modules/export-data.src.ts

```diff
@@ -379,7 +380,14 @@ setOptions({
          * @since    6.0.0
          * @requires modules/export-data
          */
-        viewData: 'View data table'
+        viewData: 'View data table',
+        /**
+         * The text for the menu item.
+         *
+         * @since    6.0.0
```

Comment:
I'm pretty sure it's not supported since v6 :) Should be `@since next` or `@since v8.1.3`